image/templates/helm/stackrox-secured-cluster/internal/cluster-config.yaml.tpl

```diff
@@ -18,7 +18,7 @@ clusterConfig:
     admissionControllerEvents: {{ ._rox.admissionControl.listenOnEvents }}
     tolerationsConfig:
       disabled: {{ ._rox.collector.disableTaintTolerations }}
-    slimCollector: {{ ._rox.collector.slimMode }}
+    slimCollector: false
```

Comment:
Do we need this line?
```suggestion
```

Reply:
My understanding is that this backwards compatibility is required for upgrading from old installations (which use slim collectors) to this version or newer which doesn't.

With that in mind, what I've hopefully achieved here is that the slim collector setting is either ignored _or_ translated when upgrading. As a result, the setting is kept in the charts and elsewhere, but is set to false for new installations and removed from docs etc since it becomes an internal-only setting (which can be fully removed after 4.6 goes out of support I think)

My testing has indicated so far that this works for old clusters configured to use slim collector

The reference to it alongside pullPolicy in the readme is definitely not needed though (pullPolicy is now Always) so I'll fix that.